content/06.model-complexity.md

```diff
@@ -1,36 +1,35 @@
-### Reducing misinterpretation of model output with statistical techniques 
+## Reducing misinterpretation of model output with statistical techniques 
 
 
 Machine learning methods are generally accompanied by a few critical assumptions. 
-First, ML methods often work best on datasets that contain equal number of samples for all categories (no "class imbalance").
-Second, the dataset is complete; all samples have measurements for all variables in the dataset (i.e., the dataset is not "sparse", meaning that it is not missing data for some samples).
+First, the training dataset contains equal number of samples for all categories (no "class imbalance").
+Second, the training dataset is complete; all samples have measurements for all variables in the dataset (i.e., the dataset is not "sparse", meaning that it is not missing data for some of the samples).
```

Comment:
Hm, would you say that these assumptions are specific to the training set?

Reply:
agree that this applies to the whole dataset.